pom.xml

```diff
@@ -1789,11 +1789,6 @@
                 <artifactId>wildfly-elytron-credential</artifactId>
                 <version>${version.org.wildfly.security.elytron}</version>
             </dependency>
-            <dependency>
-                <groupId>org.wildfly.security</groupId>
-                <artifactId>wildfly-elytron-credential-source-deprecated</artifactId>
-                <version>${version.org.wildfly.security.elytron}</version>
-            </dependency>
```

Comment:
There are also a few more dependencies on other deprecated Elytron modules that should be removed here as well:
* `wildfly-elytron-auth-server-deprecated`
* `wildfly-elytron-http-deprecated`
* `wildfly-elytron-sasl-deprecated`
* `wildfly-elytron-x500-deprecated`